CHANGELOG.md

```diff
@@ -8,7 +8,7 @@
 - Add Instant search query
 - Add `ScreenFragmentCompat` to support using the older view-based screens in the v2 navigation framework
 - Add convenience classes for creating screens for the new navigation framework
-- [In Progress: 21 Dec 2020] Instant search in Faster registration
+- [In Progress: 05 Jan 2020] Instant search in Faster registration
```

Comment:
This doesn't fall under the instant search right? It's addressing the changes for `BpPassportSheet`. Can you update the CHANGELOG accordingly, please?

Reply:
This does falls under Instant search. Vinay suggested on slack to add the changelog such that nurses understand as well. Please find the attached link in the BpPassportSheet PR. Hence updated the same.